apps/nrm/behaviours/save-application.js

```diff
@@ -14,7 +14,7 @@ const saveData = async(data) => {
 };
 
```

Comment:
I would put the save function into the models folder but maybe as part of a different PR if this PR doesn't add this function into the app